os/fs/vfs/fs_stat.c

```diff
@@ -241,3 +184,76 @@ int stat(FAR const char *path, FAR struct stat *buf)
 	set_errno(ret);
 	return ERROR;
 }
+
+/****************************************************************************
+ * Name: inode_stat
+ *
+ * Description:
+ *   The inode_stat() function will obtain information about an 'inode' in
+ *   the pseudo file system and will write it to the area pointed to by 'buf'.
+ *
+ *   The 'buf' argument is a pointer to a stat structure, as defined in
+ *   <sys/stat.h>, into which information is placed concerning the file.
+ *
+ * Input Parameters:
+ *   inode - The inode of interest
+ *   buf   - The caller provide location in which to return information about
+ *           the inode.
+ *
+ * Returned Value:
+ *   Zero (OK) returned on success.  Otherwise, a negated errno value is
+ *   returned to indicate the nature of the failure.
+ *
+ ****************************************************************************/
+
+int inode_stat(FAR struct inode *inode, FAR struct stat *buf)
+{
+	DEBUGASSERT(inode != NULL && buf != NULL);
+
+	/* Most of the stat entries just do not apply */
+
+	RESET_BUF(buf);
+
+	if (INODE_IS_SPECIAL(inode)) {
+#if defined(CONFIG_FS_NAMED_SEMAPHORES)
+		if (INODE_IS_NAMEDSEM(inode)) {
+			buf->st_mode = S_IFSEM;
+		} else
+#endif
+#if !defined(CONFIG_DISABLE_MQUEUE)
+		if (INODE_IS_MQUEUE(inode)) {
+			buf->st_mode = S_IFMQ;
+		} else
+#endif
+		;
```

Comment:
Please change to below. TizenRT usually use below style.
{
}

Reply:
Done.